client/state/reader/follows/actions.js

```diff
@@ -47,3 +52,52 @@ export function recordUnfollow( url ) {
 		} );
 	};
 }
+
+/**
+ * Returns an action object to signal that followed sites have been received.
+ *
+ * @param  {Array}  follows Follows received
+ * @return {Object} 		Action object
+ */
+export function receiveFollows( follows ) {
```

Comment:
^ Does the convention have solid reasoning behind it?  Like maybe that we sometimes want to receive outside of the normal api response (not that I can think of an instance where we do this).  

The inspiration for the question is that I would probably want to update https://github.com/Automattic/wp-calypso/pull/10856